packages/core-ui/lib/statistics/Pie/index.tsx

```diff
@@ -0,0 +1,72 @@
+/* @license Copyright 2023 @paritytech/polkadot-dashboard-ui authors & contributors
+SPDX-License-Identifier: GPL-3.0-only */
+import { useState } from "react";
+import "./index.scss";
+
+export type PieProps = {
+  title?: string;
+  titlePosition?: "top" | "right" | "bottom" | "left";
+  wrappedTitle?: boolean;
+  fixedTitle?: boolean;
+  size?: number;
+  perc?: number;
+};
+
+export const Pie = ({
+  title,
+  titlePosition = "top",
+  wrappedTitle = false,
+  fixedTitle = false,
+  size = 5,
```

Comment:
Could we have an `auto` size, that can just default to:
```
width: fit-content;
height: fit-content;
```
I guess in rectangular containers the dimensions will be bounded to maintain the same ratio.

Reply:
size depends on a param that defaults to `5` in case its not provided. This is due to the title as well, so I think it is better for the size to be requested by user rather than leaving it to surrounding space I guess